packages/discovery-provider/src/api/v1/utils/extend_notification.py

```diff
@@ -814,7 +814,7 @@ def extend_remix_contest_ending_soon(action: NotificationAction):
 
 
 def extend_artist_remix_contest_ended(action: NotificationAction):
-    data = action["data"]  # type: ignore
+    data = cast(RemixContestEndedNotification, action["data"])
```

Comment:
technically maybe we should do
`    data: ArtistRemixContestEndedNotification = action["data"]  # type: ignore`
to match existing pattern

but i can do this later

Reply:
oh gotcha ill leave the casting in cuz that's what caused the type error